compressor/filters/css_default.py

```diff
@@ -6,9 +6,10 @@
 from compressor.conf import settings
 from compressor.filters import FilterBase, FilterError
 
-URL_PATTERN = re.compile(r'url\(([^\)]+)\)')
+URL_PATTERN = re.compile(r'url\( *([\'"]?)(.+?)\1 *\)')
```

Comment:
This makes my eyes water. :-) 

*  `url(`
* Any number of spaces
* Maybe a `'` or `"`
* One or more of anything — what's the `?` doing there? 
* The matching Maybe `'` or `"`
* Any number of spaces
* The closing `)`

Is that right? 

Could you perhaps comment it for later?